helpers/Base64.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace oat\tao\helpers;
+
+/**
+ * Class Base64
+ *
+ * @package oat\tao\helpers
+ */
+class Base64
+{
+    /**
+     * @param $data
+     *
+     * @return bool
+     */
+    public function isEncoded($data): bool
```

Comment:
My apologies, I forgot to add a `static` keyword in my original suggestion.
```suggestion
    public static function isEncoded($data): bool
```